src/components/victory-scatter/point.jsx

```diff
@@ -11,7 +11,7 @@ export default class Point extends React.Component {
       x: React.PropTypes.any,
       y: React.PropTypes.any
     }),
-    labelComponent: React.PropTypes.element,
+    label: React.PropTypes.element,
```

Comment:
Hey just saw this now. I'll fix it in the yet-to-be-merged https://github.com/FormidableLabs/victory-chart/pull/117